onnxscript/function_libs/torch_lib/registration.py

```diff
@@ -2,11 +2,17 @@
 
 from __future__ import annotations
 
+import re
 from types import FunctionType
 from typing import Any, Callable, Generator, Optional
 
 import onnxscript
 
+# Regex that will match "aten::add" and "aten::add.Tensor"
```

Comment:
nit: would aten::<op_name>[.<overload>] be better? comment gives feeling the regex is specifically for add.

Reply:
Done